bin/download_metis_dir

```diff
@@ -38,7 +38,7 @@ if [[ -e /dest ]]; then
     /app/bin/metis_client $metisPath get ./ /dest/
 else
   # Ensure images are created
-  make -C docker create
+  make -C docker up
```

Comment:
Oh!  Good catch.  This can now just be `build`.  I feel stupid but I renamed `create` -> `build` for consistency because I already had some build projects, too.